src/main/java/duke/command/FindCommand.java

```diff
@@ -0,0 +1,32 @@
+package duke.command;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import duke.storage.Storage;
+import duke.task.Task;
+import duke.task.TaskList;
+import duke.ui.TextUi;
```

Comment:
I like how you organised your import statements and imported every class separately. Keep up the good work!